tests/assertions/ocp4/rhcos4-moderate-4.18.yml

```diff
@@ -1,1443 +1,1443 @@
+  e2e-moderate-master-audit-rules-privileged-commands-usernetctl:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-session-events:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-sysadmin-actions:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-time-adjtimex:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-time-clock-settime:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-time-settimeofday:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-time-stime:
+    default_result: FAIL or NOT-APPLICABLE
+    result_after_remediation: PASS or NOT-APPLICABLE
+  e2e-moderate-master-audit-rules-time-watch-localtime:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-chmod:
+    default_result: FAIL or NOT-APPLICABLE
+    result_after_remediation: PASS or NOT-APPLICABLE
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-chown:
+    default_result: FAIL or NOT-APPLICABLE
+    result_after_remediation: PASS or NOT-APPLICABLE
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-creat:
+    default_result: FAIL or NOT-APPLICABLE
+    result_after_remediation: PASS or NOT-APPLICABLE
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fchmod:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fchmodat:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fchown:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fchownat:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fremovexattr:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-fsetxattr:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-ftruncate:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-lchown:
+    default_result: FAIL or NOT-APPLICABLE
+    result_after_remediation: PASS or NOT-APPLICABLE
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-lremovexattr:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-lsetxattr:
+    default_result: FAIL
+    result_after_remediation: PASS
+  e2e-moderate-master-audit-rules-unsuccessful-file-modification-open:
```

Comment:
Looks like we still have an issue since these assertions weren't updated:

```
helpers.go:879: E2E-FAILURE: The expected default result for the e2e-moderate-master-audit-rules-unsuccessful-file-modification-open rule didn't match. Expected 'FAIL', Got 'NOT-APPLICABLE'
```